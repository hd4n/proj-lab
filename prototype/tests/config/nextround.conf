@items
ca c1
gl g1

@effects
pr p1 10

@codes
co c9

@fields
em e1
em e2

@neighbors
e1 e2

@virologists
v1 e1 c1 null p1