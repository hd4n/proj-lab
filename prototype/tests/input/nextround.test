load nextround.conf
step v1 e2
getstats v1
nextround
getstats v1