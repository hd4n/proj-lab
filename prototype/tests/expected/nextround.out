Load: sikeres
step: v1 e2

stats: v1
	fi_e1
	di_e2
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_-1
	neededNucleo_-1
	co_null
	ma_null
	ag_null
	eq_c1
	eqa_null
	ef_p1
nextround: sikeres

stats: v1
	fi_e2
	di_e2
	st_false
	res_82.3
	re_false
	ma_10
	ref_0
	neededAmino_-1
	neededNucleo_-1
	co_null
	ma_null
	ag_null
	eq_c1
	eqa_null
	ef_p1
